front/lib/api/assistant/global_agents.ts

```diff
@@ -240,19 +240,19 @@ async function _getClaudeGlobalAgent({
   };
 }
 
-async function _getMistralGlobalAgent({
+async function _getMistralSmallGlobalAgent({
   settings,
 }: {
   settings: GlobalAgentSettings | null;
 }): Promise<AgentConfigurationType> {
   const status = settings ? settings.status : "disabled_by_admin";
   return {
     id: -1,
-    sId: GLOBAL_AGENTS_SID.MISTRAL,
+    sId: GLOBAL_AGENTS_SID.MISTRAL_SMALL,
     version: 0,
     versionAuthorId: null,
-    name: "mistral",
-    description: "Mistral latest model (7B Instruct, 4k context).",
+    name: "mistral-small",
+    description: "Mistral latest model (8X7B Instruct, 32k context).",
```

Comment:
8x7b ?

Reply:
On their [doc](https://docs.mistral.ai/platform/endpoints) they use 8X7B. I'm happy to change though.